src/app/remotes/change-password/change-password.component.html

```diff
@@ -0,0 +1,10 @@
+<button
```

Comment:
The change password tab would be empty if the user does not have the permissions to change the password. So it would be better so show a message that the user does not have the permission so the tab is not empty. You can use ocxIfNotPermission for this